File: September 2022/September_22.java
class Solution {
    public String reverseWords(String s) {
        String[] arr = s.split(" ");
        StringBuilder sb = new StringBuilder();
        for(int i = 0; i < arr.length; i++){
            sb.append(reverse(arr[i]));
            if(i != arr.length - 1)
                sb.append(" ");
        }
        return sb.toString();
    }
    
    private String reverse(String s){
        StringBuilder sb = new StringBuilder();
        for(int i = s.length() - 1; i >= 0; i--)
            sb.append(s.charAt(i));
        return sb.toString();
    }
}

File: September 2022/September_22.cpp
class Solution {
public:
    string reverse(string s){
        string str = "";
        for(int i = s.length() - 1; i >= 0; i--)
            str += s[i];
        return str;
    }
    
    string reverseWords(string s) {
        string str = "", ans = "";
        for(int i = 0; i < s.length(); i++){
            if(s[i] == ' '){
                ans += reverse(str) + " ";
                str = "";
            } else
                str += s[i];
        }
        return ans + reverse(str);
    }
};
